helm/examples/pvc.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  annotations:
+    volume.beta.kubernetes.io/storage-provisioner: directpv-min-io
+    volume.kubernetes.io/selected-node: minio-k8s7
+  name: please-remove-me-cesar
```

Comment:
Do we need this example? If so, can we use https://github.com/minio/directpv/blob/master/functests/sleep.yaml instead?